scripts/build-iso

```diff
@@ -7,9 +7,10 @@ cd $(dirname $0)/..
 
 echo "Start building ISO image"
 
-HARVESTER_INSTALLER_VERSION=master
+HARVESTER_INSTALLER_VERSION=logging
```

Comment:
ditto, please remove these changes. We can keep the original values for
- build-iso
- ci
- test-integration

Reply:
reversed changes to those files